src/assets/main.scss

```diff
@@ -145,12 +145,36 @@ body {
     background: $btn-recharger-background-color !important;
   }
 }
+.is-wallet {
+  color: $btn-recharger-text-color !important;
+  border-color: $btn-recharger-border-color !important;
+  background: $color-2;
+}
+.is-wallet-active {
+  color: $btn-recharger-text-color !important;
+  border-color: $btn-recharger-border-color !important;
+  background: $color-2;
+  opacity: 0.7;
```

Comment:
@SeddikKadi 
I think it would make more sense that the **unselected** backend were with an opacity < 1 and the **selected one** full opacity, no ? The most accepted visual convention is that active stands out and inactive fades away, like an inactive button.

What do you think ?

Reply:
All good for me ! Thank you @SeddikKadi